sdk/cdn/arm-cdn/src/operations/customDomains.ts

```diff
@@ -201,42 +199,16 @@ export class CustomDomains {
    */
   enableCustomHttps(resourceGroupName: string, profileName: string, endpointName: string, customDomainName: string, options: Models.CustomDomainsEnableCustomHttpsOptionalParams, callback: msRest.ServiceCallback<Models.CustomDomain>): void;
   enableCustomHttps(resourceGroupName: string, profileName: string, endpointName: string, customDomainName: string, options?: Models.CustomDomainsEnableCustomHttpsOptionalParams | msRest.ServiceCallback<Models.CustomDomain>, callback?: msRest.ServiceCallback<Models.CustomDomain>): Promise<Models.CustomDomainsEnableCustomHttpsResponse> {
-    // #region Added default values to add backwards compatibility
```

Comment:
This part was handwritten in #11777
@qiaozha Are we sure we want to override this?

Reply:
No, we don't want to overide it. @colawwj Please change it back. Thx